### PR TITLE
fix(version): update version to 3.4.0

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Update version to 3.4.0

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/pypi-release.yml [hunk 1]
```diff
@@ -46,7 +46,6 @@ jobs:
         with:
           token: ${{ secrets.PROWLER_ACCESS_TOKEN }}
           commit-message: "chore(release): update Prowler Version to ${{ env.RELEASE_TAG }}."
-          base: master
```
Comment (jfagoagas) [inline .github/workflows/pypi-release.yml]: Please, remove this until we test it.

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: .github/workflows/pypi-release.yml [hunk 2]
```diff
@@ -46,7 +46,6 @@ jobs:
         with:
           token: ${{ secrets.PROWLER_ACCESS_TOKEN }}
           commit-message: "chore(release): update Prowler Version to ${{ env.RELEASE_TAG }}."
```
Comment (MrCloudSec) [inline .github/workflows/pypi-release.yml]:
```suggestion
          commit-message: "chore(release): update Prowler Version to ${{ env.RELEASE_TAG }}."
          base: master
```

File: .github/workflows/pypi-release.yml (hunk 1 above)
Comment (MrCloudSec) [reply .github/workflows/pypi-release.yml]: Done

**Review by jfagoagas — APPROVED** [review]
🚀

**Pull request merged**